.travis.yml

```diff
@@ -1,18 +1,32 @@
-sudo: false
+sudo: true
```

Comment:
Is there any way that we could avoid setting this? It means that we're forced to use Travis's older, slower infrastructure that doesn't support caching: https://docs.travis-ci.com/user/migrating-from-legacy/